src/ui/Sparkline/sparkline.css

```diff
@@ -1,5 +1,24 @@
-td {
+.tableui table {
```

Comment:
Some style overrides I needed to do in order to get the styles to play nicely for the nested table